stories/index.tsx

```diff
@@ -767,6 +767,14 @@ storiesOf('Griddle main', module)
       </Griddle>
     )
   })
+  .add('with Filter place-holder', () => {
```

Comment:
Can you move this down into a new Filter section of stories, similar to [the story for `settingsToggle`](https://github.com/GriddleGriddle/Griddle/blob/a3ace3027472caff209428cb30fb792bce28163f/stories/index.tsx#L1538-L1544)?

Reply:
Should I also move all the filter examples from plugins, to this section?